shell/browser/notifications/mac/notification_presenter_mac.mm

```diff
@@ -48,5 +48,18 @@
     NotificationDelegate* delegate) {
   return new CocoaNotification(delegate, this);
 }
+/*
```

Comment:
Hello @zcbenz  regarding to this:
_As far as I know for the COM server to receive events from Action Center, there must be a start menu shortcut of the app with AppUserModelID and ToastActivatorCLSID properties assigned to it, can you write the requirements in the docs?_ - You are sharing well known and widely used variant from Microsoft with usage of Shortcut and this variant actually is bad because bonds us into only one activator existing into the system
In general my words above looks next

`class DECLSPEC_UUID(COM_GUID) // Static activator with PREDEFINED UUID 
    NotificationActivator WrlSealed
    : public RuntimeClass < RuntimeClassFlags<ClassicCom>,
    INotificationActivationCallback > WrlFinal
    
    ....
    //Shortcut installation stage
    PROPVARIANT propVar;
                propVar.vt = VT_LPWSTR;
                propVar.pwszVal = const_cast<PWSTR>(AppId.c_str()); // for _In_ scenarios, we don't need a copy
                hr = propertyStore->SetValue(PKEY_AppUserModel_ID, propVar);
                if (SUCCEEDED(hr))
                {
                    propVar.vt = VT_CLSID;
                    propVar.puuid = const_cast<CLSID*>(&/*activatorID/*/__uuidof(NotificationActivator));
                    hr = propertyStore->SetValue(PKEY_AppUserModel_ToastActivatorCLSID, propVar);
                    if (SUCCEEDED(hr))
                    {
    `
    Mentioned COM_GUID should be defined only before linking, after compilation App cannot change this Activator. As result possibility to have numerous apps with different activators(this is necessary for Cold-start) is absent. As result it is not possible to have few Teams/Skypes e.t.c without recompilation their backand with purpose to have different GUID. Variant into this pull request does not use Shortcut at all and uses dynamic COM server creation. With this approach developer is able to define UUID for COM server into runtime
    
  this is part of test JS code which uses such possibilities
  `const { app } = require('electron');
 app.notificationsComServerCLSID = '{00000001-0002-0003-0405-060708090001}';//any valid value`
 
 So even with same package developer can create as much different JS instances as he wants and all instances will have their own COM server without any collisions with usage of mentioned by You static Custom activator.